modules/light-clients/08-wasm/types/types_test.go

```diff
@@ -82,6 +82,9 @@ func (suite *TypesTestSuite) SetupWasmWithMockVM() {
 	suite.coordinator = ibctesting.NewCoordinator(suite.T(), 1)
 	suite.chainA = suite.coordinator.GetChain(ibctesting.GetChainID(1))
 
+	suite.ctx = suite.chainA.GetContext().WithBlockGasMeter(storetypes.NewInfiniteGasMeter())
+	suite.store = suite.chainA.App.GetIBCKeeper().ClientKeeper.ClientStore(suite.ctx, grandpaClientID)
```

Comment:
I'd be in favour of switching var name or having another const `DefaultWasmClientID` or similar

Reply:
Makes sense, will be making another PR for this